diff -urb --binary qwt-6.1.2/designer/designer.pro qwt-6.1.2_mod/designer/designer.pro
--- qwt-6.1.2/designer/designer.pro	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/designer/designer.pro	2015-02-20 14:10:21.725895522 +0100
@@ -8,7 +8,7 @@
 ################################################################
 
 QWT_ROOT = $${PWD}/..
-QWT_OUT_ROOT = $${OUT_PWD}/..
+QWT_OUT_ROOT = $${PWD}/..
 
 include ( $${QWT_ROOT}/qwtconfig.pri )
 include ( $${QWT_ROOT}/qwtbuild.pri )
diff -urb --binary qwt-6.1.2/examples/examples.pri qwt-6.1.2_mod/examples/examples.pri
--- qwt-6.1.2/examples/examples.pri	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/examples/examples.pri	2015-02-20 14:10:21.725895522 +0100
@@ -12,7 +12,7 @@
 include( $${QWT_ROOT}/qwtbuild.pri )
 include( $${QWT_ROOT}/qwtfunctions.pri )
 
-QWT_OUT_ROOT = $${OUT_PWD}/../..
+QWT_OUT_ROOT = $${PWD}/../..
 
 TEMPLATE     = app
 
diff -urb --binary qwt-6.1.2/playground/playground.pri qwt-6.1.2_mod/playground/playground.pri
--- qwt-6.1.2/playground/playground.pri	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/playground/playground.pri	2015-02-20 14:10:21.725895522 +0100
@@ -12,7 +12,7 @@
 include( $${QWT_ROOT}/qwtbuild.pri )
 include( $${QWT_ROOT}/qwtfunctions.pri )
 
-QWT_OUT_ROOT = $${OUT_PWD}/../..
+QWT_OUT_ROOT = $${PWD}/../..
 
 TEMPLATE     = app
 
diff -urb --binary qwt-6.1.2/qwtconfig.pri qwt-6.1.2_mod/qwtconfig.pri
--- qwt-6.1.2/qwtconfig.pri	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/qwtconfig.pri	2015-02-20 14:10:21.725895522 +0100
@@ -118,7 +118,7 @@
 # Otherwise you have to build it from the designer directory.
 ######################################################################
 
-QWT_CONFIG     += QwtDesigner
+#QWT_CONFIG     += QwtDesigner
 
 ######################################################################
 # Compile all Qwt classes into the designer plugin instead
diff -urb --binary qwt-6.1.2/src/src.pro qwt-6.1.2_mod/src/src.pro
--- qwt-6.1.2/src/src.pro	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/src/src.pro	2015-02-20 14:10:21.725895522 +0100
@@ -14,7 +14,7 @@
 include( $${QWT_ROOT}/qwtbuild.pri )
 include( $${QWT_ROOT}/qwtfunctions.pri )
 
-QWT_OUT_ROOT = $${OUT_PWD}/..
+QWT_OUT_ROOT = $${PWD}/..
 
 TEMPLATE          = lib
 TARGET            = $$qwtLibraryTarget(qwt)
diff -urb --binary qwt-6.1.2/textengines/textengines.pri qwt-6.1.2_mod/textengines/textengines.pri
--- qwt-6.1.2/textengines/textengines.pri	2014-12-11 15:14:20.000000000 +0100
+++ qwt-6.1.2_mod/textengines/textengines.pri	2015-02-20 14:10:21.725895522 +0100
@@ -13,7 +13,7 @@
 include( $${QWT_ROOT}/qwtbuild.pri )
 include( $${QWT_ROOT}/qwtfunctions.pri )
 
-QWT_OUT_ROOT = $${OUT_PWD}/../..
+QWT_OUT_ROOT = $${PWD}/../..
 
 TEMPLATE  = lib
 
