--- qwtbuild.pri	2019-01-02 16:21:31.556467000 +0000
+++ qwtbuild.pri	2019-02-07 11:35:07.912813492 +0000
@@ -50,7 +50,7 @@
     #QMAKE_CXXFLAGS   *= -Wsign-conversion
     #QMAKE_CXXFLAGS   *= -Wlogical-op
     #QMAKE_CXXFLAGS   *= -Werror=format-security
-    #QMAKE_CXXFLAGS   *= -std=c++11
+    QMAKE_CXXFLAGS   *= -std=c++11
 
     # when using the gold linker ( Qt < 4.8 ) - might be
     # necessary on non linux systems too
--- qwtconfig.pri	2019-01-02 16:21:31.556467000 +0000
+++ qwtconfig.pri	2019-02-07 11:41:33.202068749 +0000
@@ -28,6 +28,8 @@
     # QWT_INSTALL_PREFIX = C:/Qwt-$$QWT_VERSION-qt-$$QT_VERSION
 }
 
+# Overwrite install prefix for use in the Hopsan build system
+QWT_INSTALL_PREFIX    = $${PWD}/../qwt
 QWT_INSTALL_DOCS      = $${QWT_INSTALL_PREFIX}/doc
 QWT_INSTALL_HEADERS   = $${QWT_INSTALL_PREFIX}/include
 QWT_INSTALL_LIBS      = $${QWT_INSTALL_PREFIX}/lib
@@ -118,7 +120,7 @@
 # Otherwise you have to build it from the designer directory.
 ######################################################################
 
-QWT_CONFIG     += QwtDesigner
+#QWT_CONFIG     += QwtDesigner
 
 ######################################################################
 # Compile all Qwt classes into the designer plugin instead
@@ -161,7 +163,7 @@
 
 macx:!static:CONFIG(qt_framework, qt_framework|qt_no_framework) {
 
-    QWT_CONFIG += QwtFramework
+#    QWT_CONFIG += QwtFramework
 }
 
 ######################################################################
