Index: qwt_abstract_scale_draw.cpp
===================================================================
--- qwt-6.1.0/src/qwt_abstract_scale_draw.cpp	(revision 1758)
+++ qwt-6.1.0/src/qwt_abstract_scale_draw.cpp	(revision 1759)
@@ -369,9 +369,6 @@
 */
 QwtText QwtAbstractScaleDraw::label( double value ) const
 {
-    if ( qFuzzyCompare( value + 1.0, 1.0 ) )
-        value = 0.0; 
-
     return QLocale().toString( value );
 }
 
--- qwt-6.1.0/qwtconfig.pri	2014-12-08 09:59:44.917220264 +0100
+++ qwt-6.1.0/qwtconfig.pri	2014-12-08 10:00:04.921221232 +0100
@@ -116,7 +116,7 @@
 # Otherwise you have to build it from the designer directory.
 ######################################################################
 
-QWT_CONFIG     += QwtDesigner
+#QWT_CONFIG     += QwtDesigner
 
 ######################################################################
 # Compile all Qwt classes into the designer plugin instead

